packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -1075,20 +1104,20 @@ class _RenderDecoration extends RenderBox {
       + contentPadding.bottom
       + densityOffset.dy,
     );
-    final double minContainerHeight = decoration.isDense || expands
-      ? 0.0
-      : kMinInteractiveDimension + densityOffset.dy;
+    final double minContainerHeight = decoration.isDense
```

Comment:
This expression and the following one might be a little easier to read if allowed the lines to get a little wider.  Doing it this way makes the similarity of these statements easier to see.
```dart
final double minContainerHeight =  expands || decoration.isDense || decoration.contentPadding != null
  ? 0.0
  : kMinInteractiveDimension + densityOffset.dy;
```